blazor/treegrid/clipboard.md

```diff
@@ -384,4 +384,16 @@ N> To perform paste functionality, it requires the selection `Mode` to be `Cell`
 
 ### Limitations of Paste Functionality
 
-* Since the string values are not parsed to number and date type, so when the copied string type cells are pasted to number type cells then it will be displayed as **NaN**. For date type cells, when the copied string format cells are pasted to date type cells then it will be displayed as an **empty cell**.
\ No newline at end of file
+* Since the string values are not parsed to number and date type, so when the copied string type cells are pasted to number type cells then it will be displayed as **NaN**. For date type cells, when the copied string format cells are pasted to date type cells then it will be displayed as an **empty cell**.
+
+## Clipboard events
+
+Clipboard events are events that are triggered when performing a copy or paste action on the tree grid. 
```

Comment:
Clipboard events are triggered when performing a copy or paste action on the tree grid.

Reply:
Resolved by changing the sentence.